renderer/index.js

```diff
@@ -153,3 +153,10 @@ ipc.on('removeRows', function() {
 ipc.on('removeColumns', function() {
   hotController.removeColumns();
 });
+
+ipc.on('freezeRows', function(){
```

Comment:
This event hander should be updated to expect an event called `'freeze'` to match with the event you're raising [here](https://github.com/theodi/comma-chameleon/pull/169/files?w=1#diff-b8bbf1b64cab439cc1c4ead74d50d1aeR203)